src/Boolean.ts

```diff
@@ -91,17 +92,33 @@ export const SemigroupAll: semigroup.Semigroup<boolean> = semigroup.booleanAll
  *
  * @example
  * import { SemigroupAny } from '@fp-ts/core/Boolean'
- * import { pipe } from '@fp-ts/core/Function'
  *
  * assert.deepStrictEqual(SemigroupAny.combine(true, true), true)
  * assert.deepStrictEqual(SemigroupAny.combine(true, false), true)
+ * assert.deepStrictEqual(SemigroupAny.combine(false, true), true)
  * assert.deepStrictEqual(SemigroupAny.combine(false, false), false)
  *
  * @category instances
  * @since 1.0.0
  */
 export const SemigroupAny: semigroup.Semigroup<boolean> = semigroup.booleanAny
 
+/**
+ * `boolean` semigroup under disjunction.
+ *
+ * @example
+ * import { SemigroupExclusiveAny } from '@fp-ts/core/Boolean'
+ *
+ * assert.deepStrictEqual(SemigroupExclusiveAny.combine(true, true), false)
+ * assert.deepStrictEqual(SemigroupExclusiveAny.combine(true, false), true)
+ * assert.deepStrictEqual(SemigroupExclusiveAny.combine(false, true), true)
+ * assert.deepStrictEqual(SemigroupExclusiveAny.combine(false, false), false)
+ *
+ * @category instances
+ * @since 1.0.0
+ */
+export const SemigroupExclusiveAny: semigroup.Semigroup<boolean> = semigroup.booleanExclusiveAny
```

Comment:
What do you think of `SemigroupXor`? It seems shorter and more easily recognizable

Reply:
Agree - it will be better. I will change